PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -310,13 +298,20 @@ def miniAOD_customizeCommon(process):
         process.heepIDVarValueMaps.elesMiniAOD = cms.InputTag('reducedEgamma','reducedGedGsfElectrons')
         #force HEEP to use miniAOD (otherwise it'll detect the AOD)
         process.heepIDVarValueMaps.dataFormat = cms.int32(2)
-        
 
+        #add the HEEP trk isol to the slimmed electron
+        process.slimmedElectrons.modifierConfig.modifications[0].electron_config.heepV70TrkPtIso = cms.InputTag("heepIDVarValueMaps","eleTrkPtIso")
+
+    #now put them all in the pat electron
+    process.patElectrons.electronIDSources=cms.PSet()
+    for egid in process.egmGsfElectronIDs.physicsObjectIDs:
+        setattr(process.patElectrons.electronIDSources,egid.idDefinition.idName.value(),cms.InputTag('egmGsfElectronIDs:'+egid.idDefinition.idName.value()))  
```

Comment:
apparently this logic belongs in PhysicsTools/SelectorUtils/python/tools/vid_id_tools.py
the miniAOD_tools should not be exposed so much to the details of VID implementation

Reply:
was already been done in there, so can just delete